test/run/test/exec-args.test.ts

```diff
@@ -0,0 +1,13 @@
+import { describe, expect, it } from 'vitest'
+import { runVitestCli } from '../../test-utils'
+
+describe('exec-args', async () => {
```

Comment:
These tests work and look good, but require quite much setup, e.g. 3x `vitest.config.ts`. Instead, you could create test cases to `fixtures/` and use `runVitest` (not cli) from test utils. Something like `test/env-mixed/test/mixed-environments.test.ts` does.

This might work:

```ts
import { expect, test } from 'vitest'
import { runVitest } from '../../test-utils'

test.each(['forks', 'threads', 'vmThreads'] as const)('should pass execArvs to { pool: "%s" } ', async (pool) => {
  const vitest = await runVitest({
    include: ['fixtures/exec-args.test.ts'],
    pool,
    poolOptions: {
      [pool]: {
        execArgv: [
          '--hash-seed=1',
          '--random-seed=1',
          '--no-opt',
        ],
      },
    },
  })

  expect(vitest.stdout).toContain('✓ exec-args-fixtures')
})

```

Reply:
Thanks that worked perfectly!